### DISCUSSION TITLE
3.8.5

### DISCUSSION THREAD

**Discussion by gsemet** [Guake/guake#2040] [Announcements]
## What's Changed
* Fixed readthedoc system
* Fix system font only being applied to last tab by @Davidy22 in https://github.com/Guake/guake/pull/1954
* Use keycode for keybindings by @vantu5z in https://github.com/Guake/guake/pull/1953
* Replace ｜ with non unicode dependent | by @Davidy22 in https://github.com/Guake/guake/pull/1966
* Update news file by @Davidy22 in https://github.com/Guake/guake/pull/1951
* Add pbr fallback by @Davidy22 in https://github.com/Guake/guake/pull/1964
* Fix vte spawn_sync runtime check failed by @mlouielu in https://github.com/Guake/guake/pull/1974
* Add `GUAKE_ENABLE_WAYLAND` environment variable by @VitalyAnkh in https://github.com/Guake/guake/pull/1975
* Add instructions for PPA install by @Davidy22 in https://github.com/Guake/guake/pull/1961
* Switch default for hide tab bar if only one tab to disabled by @Davidy22 in https://github.com/Guake/guake/pull/1983
* Changed default keybindig to toggle hide on lose focus. Fix #1719 by @jcfstabile in https://github.com/Guake/guake/pull/1988
* Fix module gi not found when make run  by @jcfstabile in https://github.com/Guake/guake/pull/1989
* Revert "Fix module gi not found when make run " by @mlouielu in https://github.com/Guake/guake/pull/1992
* Drop pbr by @eli-schwartz in https://github.com/Guake/guake/pull/1991
* Bump pipenv from 2021.5.29 to 2022.1.8 by @dependabot in https://github.com/Guake/guake/pull/2005
* Fix upload to pypi test by @Davidy22 in https://github.com/Guake/guake/pull/2002
* Fix translation file inclusion by @Davidy22 in https://github.com/Guake/guake/pull/2004
* feature: --is-visible option #1926 by @rjeb in https://github.com/Guake/guake/pull/2008
* Reinstate double click to open tabs by @Davidy22 in https://github.com/Guake/guake/pull/2010
* Fix deepsource complaints by @Davidy22 in https://github.com/Guake/guake/pull/2009
* ci: enable upload to pypi by @Davidy22 in https://github.com/Guake/guake/pull/2007
* Generate release notes for 3.8.2 by @Davidy22 in https://github.com/Guake/guake/pull/2012
* Update translation from Weblate by @gsemet in https://github.com/Guake/guake/pull/2013
* Revert "doc: update NEWS.rst" by @Davidy22 in https://github.com/Guake/guake/pull/2014
* ci: no pipeline on pull_request by @gsemet in https://github.com/Guake/guake/pull/2016
* Revert "ci: no pipeline on pull_request" by @Davidy22 in https://github.com/Guake/guake/pull/2018
* Fix arbitrary execution via dbus security flaw by @Davidy22 in https://github.com/Guake/guake/pull/2017
* Eliminate redundant terminal spawning by @Davidy22 in https://github.com/Guake/guake/pull/2020
* Generate release notes for 3.8.2 by @Davidy22 in https://github.com/Guake/guake/pull/2024

## New Contributors
* @VitalyAnkh made their first contribution in https://github.com/Guake/guake/pull/1975
* @jcfstabile made their first contribution in https://github.com/Guake/guake/pull/1988
* @rjeb made their first contribution in https://github.com/Guake/guake/pull/2008

**Full Changelog**: https://github.com/Guake/guake/compare/3.8.1...3.8.5

<hr /><em>This discussion was created from the release <a href='https://github.com/Guake/guake/releases/tag/3.8.5'>3.8.5</a>.</em>

**Comment by Davidy22** [comment]
Why is the whole changelog from 3.8.2 being folded in here, why's github doing this from 3.8.1 to now instead of from 3.8.4 to 3.8.5

**Reply by gsemet** [reply]
3.8.2, 3.8.3 and 3.8.5 does not exist on pypi, thus we can kind of "discard" them (like if they never existed). I can also remove the tags, since they do not correspond to release we want to see in the nature.

**Reply by Davidy22** [reply]
eh, sure
